### PR TITLE
Update structs and types

### PR DESCRIPTION
**Pull request by ydakuka**
I checked the available structs.

### CODE REVIEW

**Review by rwehresmann — CHANGES_REQUESTED** [review]
@ydakuka Please keep `error_response.rb` as snakecase. My idea with the `struct` package is: 

1) What we'll inform in a request we keep as camelcase because is how Sumsub API is already expecting to receive;
2) The responses we'll consume in our Ruby applications we keep as snake case because is ruby style.

And could you please reference in the PR description the relevant API docs for each structure? Just to facilitate the CR and see if everything matches the docs.

**Comment by ydakuka** [comment]
I have added aliases for camelCase attributes. I think this is exactly what we need.

**Comment by rwehresmann** [comment]
> I have added aliases for camelCase attributes. I think this is exactly what we need.

Why, exactly? If you could explain to me why you need that, I can think about it.  The way I see we're adding something that increases maintenance just because of personal preference in naming variables. I prefer to keep it standard.

**Comment by ydakuka** [comment]
> If you could explain to me why you need that, I can think about it.

The format for defining attributes is the same for all structs. The attributes of the structs are the same as what we get from the response. You can read attribute values in both camelCase and snakecase styles.

>  The way I see we're adding something that increases maintenance just because of personal preference in naming variables.

I agree, but I don’t know of a simpler format for aliases.

**Comment by rwehresmann** [comment]
It's a matter of input and output: Sumsub API expects the input with camelcase variables; Sumsub API output are consumed by us in our ruby applications, where we use the snake case variables. I don't see a problem with this, so I'll keep it that way.

If there is any change/problem with the structures themselves (attribute missing, spelling, etc) feel free to rebase this PR (or create a new one) with the relevant modifications, providing Sumsbub API docs relevant links with the updated structs that support those modifications.

**Comment by ydakuka** [comment]
Ok, done. But it's very strange that some structs use camelCase, while others use snake_case.

File: lib/sumsub/struct/applicant.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ class Applicant < BaseStruct
 
       attribute? :sourceKey, Types::String
       attribute? :email, Types::String
+      attribute? :phone, Types::String
```
Comment (ydakuka) [inline lib/sumsub/struct/applicant.rb:10]: https://developers.sumsub.com/api-reference/#request-body

File: lib/sumsub/struct/applicant_update.rb [hunk 2]
```diff
@@ -7,11 +7,14 @@ class ApplicantUpdate < BaseStruct
...
-      attribute? :sourceKey, Types::String
       attribute? :email, Types::String
+      attribute? :phone, Types::String
+      attribute? :sourceKey, Types::String
+      attribute? :type, Types::String
       attribute? :lang, Types::String
+      attribute? :questionnaires, Types::Array.of(Types::Hash)
       attribute? :metadata, Types::Array.of(Types::Hash)
-      attribute? :type, Types::String
+      attribute? :deleted, Types::Bool
```
Comment (ydakuka) [inline lib/sumsub/struct/applicant_update.rb:16]: https://developers.sumsub.com/api-reference/#request-body-2

File: lib/sumsub/struct/info.rb [hunk 3]
```diff
@@ -18,7 +18,6 @@ class Info < BaseStruct
       attribute? :stateOfBirth, Types::String
       attribute? :country, Types::String
       attribute? :nationality, Types::String
-      attribute? :phone, Types::String
```
Comment (ydakuka) [inline lib/sumsub/struct/info.rb:21]: https://developers.sumsub.com/api-reference/#fixedinfo-and-info-attributes

File: lib/sumsub/types.rb [hunk 4]
```diff
@@ -11,22 +11,21 @@ module Types
...
         :CONTRACT,
-        :RESIDENCE_PERMIT,
-        :EMPLOYMENT_CERTIFICATE,
         :DRIVERS_TRANSLATION,
         :INVESTOR_DOC,
         :VEHICLE_REGISTRATION_CERTIFICATE,
         :INCOME_SOURCE, # Proof of income
         :PAYMENT_METHOD,
+        :BANK_CARD,
+        :COVID_VACCINATION_FORM,
```
Comment (ydakuka) [inline lib/sumsub/types.rb:28]: https://developers.sumsub.com/api-reference/#supported-document-types

File: lib/sumsub/types.rb [hunk 5]
```diff
@@ -36,6 +35,7 @@ module Types
       .enum(
         :FRONT_SIDE,
         :BACK_SIDE,
+        nil,
```
Comment (ydakuka) [inline lib/sumsub/types.rb:39]: https://developers.sumsub.com/api-reference/#request-metadata-body-part-fields

**Review by rwehresmann — APPROVED** [review]

**Comment by ydakuka** [comment]
Thanks.

**Pull request merged**